Modules/Loadable/VolumeRendering/MRML/vtkMRMLShaderPropertyNode.cxx

```diff
@@ -147,7 +147,7 @@ vtkMRMLStorageNode* vtkMRMLShaderPropertyNode::CreateDefaultStorageNode()
   vtkMRMLScene* scene = this->GetScene();
   if (scene == nullptr)
     {
-    vtkErrorMacro("CreateDefaultStorageNode failed: scene is invalid");
+    vtkErrorMacro(tr("CreateDefaultStorageNode failed: scene is invalid"));
```

Comment:
1. Log messages must not be translated - https://github.com/Slicer/SlicerLanguagePacks/blob/main/DevelopersManual.md#identifying-translatable-strings
2. We don't have infarstructure yet for translating messages in VTK files.
```suggestion
    vtkErrorMacro("CreateDefaultStorageNode failed: scene is invalid");
```